CHANGELOG.md

```diff
@@ -3,9 +3,14 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+All official releases can be found on this repository's [releases page](https://github.com/ChartBoost/chartboost-mediation-android-adapter-applovin/releases).
+
 ### 5.12.6.1.0
 - This version of the adapter has been certified with AppLovin SDK 12.6.1.
 
+### 4.12.6.1.0
+- This version of the adapter has been certified with AppLovin SDK 12.6.1.
+
```

Comment:
Should this be below the 5.x entries?

Reply:
we are doing them directly under their 5.x counterpart.

Reply:
it's a bit jarring. why not just separate sections—1 for 4.x and 1 for 5.x?